integration-tests/rest-client-reactive-stork/src/test/java/io/quarkus/it/rest/reactive/stork/RestClientReactiveStorkTest.java

```diff
@@ -0,0 +1,72 @@
+package io.quarkus.it.rest.reactive.stork;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
+import static io.restassured.RestAssured.when;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.response.Response;
+
+@QuarkusTest
+public class RestClientReactiveStorkTest {
+
+    static final String SLOW_RESPONSE = "hello, I'm a slow server";
+    static final String FAST_RESPONSE = "hello, I'm a fast server";
+    static WireMockServer slowServer;
```

Comment:
Would it not be preferable to use what is described in https://quarkus.io/guides/rest-client#using-a-mock-http-server-for-tests?

The reason is that the linked way starts Wiremocks before Quarkus and can affect the build via properties

Of course we would need 2 since you are starting 2 wiremocks

Reply:
done